src/buttons/radio.ts

```diff
@@ -57,8 +57,8 @@ export class NgbRadioGroup implements ControlValueAccessor {
 }
 
 
-@Directive({selector: 'label.btn'})
-export class NgbActiveLabel {
+@Directive({selector: '[ngbButtonLabel]', host: {'[class.btn]': 'true'}})
```

Comment:
This would potentially overwrite classes that people would like to set on the `<label>` element, so I would prefer to leave it as-is.